Platform/AlderlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -571,6 +624,12 @@ BoardInit (
 
     //Initialize and setup MeMeasureboot
     MeMeasuredBootInit();
+#if FixedPcdGetBool(PcdAzbSupport)
+    // Workaround for specific hardware design of CRB platform:
+    // USB instance 0 is mapped to eMMC, so it must be initialized
+    // and deinitialized to enable USB flash drive detection.
+    UsbDevInstanceInitDeinit();
```

Comment:
If USB boot option is not in the boot option list or not the first boot option, it is not necessary to apply this option. It would impact boot performance. 
Not sure if we could narrow down which settings (registers) are necessary to WA the HW issue instead of calling USB init and de-init.

Reply:
I am ok if this is just a temp fix to unblock users.